### PR TITLE
Add linters and dependabot integration

### PR DESCRIPTION
**Pull request by mestery**
Signed-off-by: Kyle Mestery <mestery@mestery.com>

### CODE REVIEW

File: openoffload/cpp/framework/Dockerfile [hunk 1]
```diff
@@ -1,23 +1,29 @@
...
 COPY openoffload.proto /home/protos/
-RUN ls ../../../protos
-RUN mkdir -p bin
-RUN mkdir -p obj
-RUN mkdir -p log
-RUN make clean
-RUN make HOME=/home/grpc all
+
+RUN ls ../../../protos && \
+    mkdir -p bin && \
```
Comment (glimchb) [inline openoffload/cpp/framework/Dockerfile:23]:
```suggestion
    mkdir -p {bin,obj,log} &&
```

File: openoffload/cpp/framework/Dockerfile [hunk 2]
```diff
@@ -1,23 +1,29 @@
 FROM grpcbuild:v1beta1
-#
 USER root
 ENV PATH="/home/grpc/local/bin:${PATH}"
-RUN mkdir -p /home/grpc/local/tests
-RUN mkdir -p /home/grpc/local/tests/src/
-RUN mkdir -p /home/grpc/local/tests/include/
-RUN mkdir -p /home/grpc/local/tests/ssl/
+
+RUN mkdir -p /home/grpc/local/tests && \
```
Comment (glimchb) [inline openoffload/cpp/framework/Dockerfile:5]:
```suggestion
RUN mkdir -p /home/grpc/local/tests/{src,include,ssl}
```

File: openoffload/cpp/framework/build/Dockerfile [hunk 3]
```diff
@@ -1,44 +1,53 @@
...
-RUN ./configure
-RUN make
-#
-# Install in default location /usr/local
-#
-RUN make install
+
+RUN ./configure && \
+    make && \
+    RUN make install
```
Comment (glimchb) [inline openoffload/cpp/framework/build/Dockerfile:49]:
```suggestion
    make install
```

File: openoffload/cpp/framework/Dockerfile [hunk 4]
```diff
@@ -2,7 +2,7 @@ FROM grpcbuild:v1beta1
 USER root
 ENV PATH="/home/grpc/local/bin:${PATH}"
 
-RUN mkdir -p /home/grpc/local/tests/{src,include,ssl}
+RUN mkdir -p /home/grpc/local/tests/[src,include,ssl]
```
Comment (glimchb) [inline openoffload/cpp/framework/Dockerfile:5]: should be `{}`

File: openoffload/cpp/framework/Dockerfile [hunk 5]
```diff
@@ -17,7 +17,7 @@ RUN mkdir -p /home/protos
 COPY openoffload.proto /home/protos/
 
 RUN ls ../../../protos && \
-    mkdir -p {bin,obj,log} && \
+    mkdir -p [bin,obj,log] && \
```
Comment (glimchb) [inline openoffload/cpp/framework/Dockerfile:20]: should be `{}`

File: openoffload/cpp/framework/Dockerfile (hunk 4 above)
Comment (mestery) [reply openoffload/cpp/framework/Dockerfile]: Also need to be run with `bash -c`. Fixed in latest commit.

File: RELEASE_NOTES.md [hunk 6]
```diff
@@ -4,107 +4,106 @@
...
--  Promote v1beta1 to v1 as the first official release.
--  Add Integration FAQ to help hardware developers understand details of the solution
-
+* Promote v1beta1 to v1 as the first official release.
+* Add Integration FAQ to help hardware developers understand details of the solution
 
 ## Release v1beta1
 
-### Bug Fixes
+### v1beta1 Bug Fixes
```
Comment (glimchb) [inline RELEASE_NOTES.md:16]: ?

File: RELEASE_NOTES.md [hunk 7]
```diff
@@ -4,107 +4,106 @@
...
 
-- For python tests, updated the ssl certificate since the original had expired. Changed expiry to 10 years.
+* For python tests, updated the ssl certificate since the original had expired. Changed expiry to 10 years.
 
-### Feature Changes
+### v1beta1 Feature Changes
```
Comment (glimchb) [inline RELEASE_NOTES.md:20]: ?

**Review by glimchb — APPROVED** [review]
a lot of linter fixes, great work @mestery !

**Pull request merged**